From d48e001d4952b4c75f29375d0d122925804579ba Mon Sep 17 00:00:00 2001
From: hepengpx <pengpengx.he@intel.com>
Date: Fri, 16 Jan 2026 00:58:48 -0700
Subject: [PATCH 62/76] i2c: media: fix cov issue

check return value of v4l2_ctrl_handler_init;
change the type of phy_port to int;

Signed-off-by: hepengpx <pengpengx.he@intel.com>
Signed-off-by: zouxiaoh <xiaohong.zou@intel.com>
---
 6.18.0/drivers/media/pci/intel/ipu6/ipu6-isys-mcd-phy.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/6.18.0/drivers/media/pci/intel/ipu6/ipu6-isys-mcd-phy.c b/6.18.0/drivers/media/pci/intel/ipu6/ipu6-isys-mcd-phy.c
index f18e6fa..ff29130 100644
--- a/6.18.0/drivers/media/pci/intel/ipu6/ipu6-isys-mcd-phy.c
+++ b/6.18.0/drivers/media/pci/intel/ipu6/ipu6-isys-mcd-phy.c
@@ -644,7 +644,8 @@ static int ipu6_isys_driver_port_to_phy_port(struct ipu6_isys_csi2_config *cfg)
 #if IS_ENABLED(CONFIG_VIDEO_INTEL_IPU_USE_PLATFORMDATA)
 static int ipu6_isys_mcd_phy_config(struct ipu6_isys *isys, struct ipu6_isys_csi2_config *cfg)
 {
-	unsigned int phy_port, phy_id;
+	unsigned int phy_id;
+	int phy_port;
 	void __iomem *phy_base;
 	struct ipu6_bus_device *adev = to_ipu6_bus_device(&isys->adev->auxdev.dev);
 	struct ipu6_device *isp = adev->isp;
-- 
2.43.0

